### ISSUE TITLE
Can't add change the billing address

### ISSUE THREAD

**Issue by harshitmahendra** [hallme/woo-address-book#63]
The plugin only changes the Shipping address by clicking on Make Primary Button!
Can't change the billing address??

**Comment by matt-h** [comment]
That is correct, this plugin currently only works with Shipping Addresses in WooCommerce.

We've been hoping to expand the functionality to also work with Billing Addresses at some point, but so far no work has been made on progress towards that.

**Comment by harshitmahendra** [comment]
any progress?? 
can I contribute?

**Comment by matt-h** [comment]
Hi @harshitmahendra 

No, it probably won't be for quite awhile before we get to tackling something like this as we haven't needed it yet.

We are open to pull requests if you want to take a stab at it.

**Comment by ezkops** [comment]
Does anyone have a solution for this, if I enable this ' Force shipping to the customer billing address' the plugin should work?
